install/installation-linux.md

```diff
@@ -0,0 +1,525 @@
+
+    ```bash
+    apt update
+    ```
+
+1.  Install TimescaleDB:
+
+    ```bash
+    apt install timescaledb-2-postgresql-15
+    ```
+
+    <highlight type="note">
+    If you want to install a specific version of TimescaleDB, instead of the
+    most recent, you can specify the version like this:
+    `apt-get install timescaledb-2-postgresql-12='2.6.0*' timescaledb-2-loader-postgresql-12='2.6.0*'`
+
+    You can see the full list of TimescaleDB releases by visiting our
+    [releases page][releases-page]. Note that older versions of TimescaleDB
+    don't always support all the OS versions listed above.
+    </highlight>
+
+When you have completed the installation, you need to configure your database so
+that you can use it. The easiest way to do this is to run the `timescaledb-tune`
+script, which is included with the `timescaledb-tools` package. Run the
+`timescaledb-tune` script using the `sudo timescaledb-tune` command. For more
+information, see the [configuration][config] section.
+
+</procedure>
+
+</Tab>
+
+<Tab title="Red Hat">
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on Red Hat-based systems
+
+1.  At the command prompt, as root, add the PostgreSQL third party repository
+    to get the latest PostgreSQL packages:
+    <terminal>
+
+    <tab label='Red Hat'>
+
+    ```bash
+    yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-$(rpm -E %{rhel})-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    ```
+
+    </tab>
+
+    <tab label="Fedora">
+
+    ```bash
+    yum install https://download.postgresql.org/pub/repos/yum/reporpms/F-$(rpm -E %{fedora})-x86_64/pgdg-fedora-repo-latest.noarch.rpm
+    ```
+
+    </tab>
+    </terminal>
+1.  Create the Timescale repository:
+    <terminal>
+
+    <tab label='Red Hat'>
+
+    ```bash
+    tee /etc/yum.repos.d/timescale_timescaledb.repo <<EOL
+    [timescale_timescaledb]
+    name=timescale_timescaledb
+    baseurl=https://packagecloud.io/timescale/timescaledb/el/$(rpm -E %{rhel})/\$basearch
+    repo_gpgcheck=1
+    gpgcheck=0
+    enabled=1
+    gpgkey=https://packagecloud.io/timescale/timescaledb/gpgkey
+    sslverify=1
+    sslcacert=/etc/pki/tls/certs/ca-bundle.crt
+    metadata_expire=300
+    EOL
+    ```
+
+    </tab>
+
+    <tab label="Fedora">
+
+    ```bash
+    tee /etc/yum.repos.d/timescale_timescaledb.repo <<EOL
+    [timescale_timescaledb]
+    name=timescale_timescaledb
+    baseurl=https://packagecloud.io/timescale/timescaledb/el/8/$basearch
+    repo_gpgcheck=1
+    gpgcheck=0
+    enabled=1
+    gpgkey=https://packagecloud.io/timescale/timescaledb/gpgkey
+    sslverify=1
+    sslcacert=/etc/pki/tls/certs/ca-bundle.crt
+    metadata_expire=300
+    EOL
+    ```
+
+    </tab>
+    </terminal>
+1.  Update your local repository list:
+
+    ```bash
+    yum update
+    ```
+
+1.  Install TimescaleDB:
+
+    ```bash
+    yum install timescaledb-2-postgresql-14
+    ```
+
+    <!-- hack until we have bandwidth to rewrite this linting rule -->
```

Comment:
Add it to the [standing issue](https://github.com/timescale/docs/issues/1369) ... ?